github/orgs_test.go

```diff
@@ -109,6 +109,26 @@ func TestOrganizationsService_Get_invalidOrg(t *testing.T) {
 	testURLParseError(t, err)
 }
 
+func TestOrganizationsService_GetByID(t *testing.T) {
+	setup()
+	defer teardown()
+
+	mux.HandleFunc("/organizations/1", func(w http.ResponseWriter, r *http.Request) {
+		testMethod(t, r, "GET")
+		fmt.Fprint(w, `{"id":1, "login":"l", "url":"u", "avatar_url": "a", "location":"l"}`)
+	})
+
+	org, _, err := client.Organizations.GetByID(context.Background(), 1)
+	if err != nil {
+		t.Errorf("Organizations.GetByID returned error: %v", err)
```

Comment:
Change `t.Errorf` to `t.Fatalf` here. If we get non-nil error, it doesn't make sense to continue and try to look at result.

(If you don't mind, make a similar improvement to `TestUsersService_GetByID` and `TestRepositoriesService_GetByID`.)